app/controllers/api/site_configuration_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::SiteConfigurationController < Api::ApplicationController
+  before_action :set_site_configuration, only: %i[index create]
+
+  skip_after_action :verify_policy_scoped
+  after_action :verify_authorized
```

Comment:
isn't this only for index?

Reply:
the reason these are here is to override the same in the parent, which specifies actions that dont exist in this subclass and cause errors